### PR TITLE
Requesting For [https://edma.atlassian.net/browse/KIR-89](url)

### PR DESCRIPTION
**Pull request by SelcukEdma**
Requesting Approval Dropdown Menu for RequestAssets.
Note: Added extra Css for `react-select and react-window` in order to tweak the styling.

### CODE REVIEW

File: src/Components/Scenes/RequestAssets/RequestingFor/RequestingFor.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    width: 'max-content',
+    textTransform: 'uppercase',
+    marginBottom: 8
+  }
+});
+
+const customStyles = {
+  option: (provided, state) => ({
+    ...provided,
+    fontSize: 12
```
Comment (davidkliewer-kirby) [inline src/Components/Scenes/RequestAssets/RequestingFor/RequestingFor.js]: @sfowles font sizes, and colors should all be using Design tokens, no?

Comment (SelcukEdma) [reply src/Components/Scenes/RequestAssets/RequestingFor/RequestingFor.js]: Not fontSize but color yes.

File: src/Components/Scenes/RequestAssets/RequestAsset/RequestAsset-Container.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
 import { connect } from 'react-redux';
-import RequestAsset from './RequestAsset';
+import RequestTable from './RequestAsset';
```
Comment (ghost) [inline src/Components/Scenes/RequestAssets/RequestAsset/RequestAsset-Container.js]: nitpick: should this be `RequestAsset` instead? I don't see anything referenced as `RequestTable` anywhere else?

**Review by ghost — COMMENTED** [review]
Little bug I found: for some reason, the user under Sonia doesn't have a name?
Screen Shot 2019-11-22 at 1 01 31 PM

Overall,  I'm concerned about some of the naming conventions here. Maybe we can hop on a call to discuss futher?

**Comment by SelcukEdma** [comment]
> Little bug I found: for some reason, the user under Sonia doesn't have a name?
> Screen Shot 2019-11-22 at 1 01 31 PM
> 
> Overall, I'm concerned about some of the naming conventions here. Maybe we can hop on a call to discuss futher?

I know that name doesn’t exist in there. If you take a look closely You will find some names  includes`?` `!` . Which is coming from an API. Our backend team cant solve it. Cuz it that Api get from another team We open a ticket for this waiting them to update it. 
Screen Shot 2019-11-22 at 12 37 45 PM
Will be free after 2:30pm we can talk about the name convention

File: src/Components/Scenes/RequestAssets/RequestAsset/RequestAsset-Container.js (hunk 2 above)
Comment (SelcukEdma) [reply src/Components/Scenes/RequestAssets/RequestAsset/RequestAsset-Container.js]: Changed

File: src/Components/Scenes/RequestAssets/RequestingFor/RequestingFor.js (hunk 1 above)
Comment (SelcukEdma) [reply src/Components/Scenes/RequestAssets/RequestingFor/RequestingFor.js]: added fontsize

File: src/assets/styles/components/SideBar.scss [hunk 3]
```diff
@@ -9,6 +9,6 @@
   position: fixed;
   width: 31.21%;
   height: 100vh;
-  background: rgba(0, 0, 0, 0.09);
+  background: #e6e6e6;
```
Comment (ghost) [inline src/assets/styles/components/SideBar.scss]: nitpick: would this also be a design token?

**Review by ghost — COMMENTED** [review]
Sweetness! Only nitpick is that in `Api/requestAsset.js` the api call is named `requestAssetApiCall`, which is still vague. Something like `getAllEmployees` or `getEmployees` would be sufficient.

File: src/assets/styles/components/SideBar.scss (hunk 3 above)
Comment (SelcukEdma) [reply src/assets/styles/components/SideBar.scss]: `#e6e6e6` is missing. Needed an update for design token. I will contacting Scott to update the right color.

**Comment by SelcukEdma** [comment]
> Sweetness! Only nitpick is that in `Api/requestAsset.js` the api call is named `requestAssetApiCall`, which is still vague. Something like `getAllEmployees` or `getEmployees` would be sufficient.

Alright changed the names to `getEmployees`

File: src/assets/styles/components/SideBar.scss (hunk 3 above)
Comment (robertwollny-pnmac) [reply src/assets/styles/components/SideBar.scss]: --color-g-200

Comment (robertwollny-pnmac) [reply src/assets/styles/components/SideBar.scss]: The functionality looks good. I dropped in the background color for your from Blocks

Comment (SelcukEdma) [reply src/assets/styles/components/SideBar.scss]: @robertwollny-pnmac  I tried that g-200 it seems color doesn’t match with the design specks

**Review by ghost — DISMISSED** [review]

File: src/assets/styles/components/SideBar.scss (hunk 3 above)
Comment (davidkliewer-kirby) [reply src/assets/styles/components/SideBar.scss]: If it doesn't match the design specs, I'd add a // TODO note in the code base, and make sure @sfowles  is aware.

**Review by davidkliewer-kirby — DISMISSED** [review]

**Review by davidkliewer-kirby — APPROVED** [review]

**Review by robertwollny-pnmac — APPROVED** [review]

**Pull request merged**